.gitignore

```diff
@@ -1,7 +1,13 @@
 *.pyc
-proto/*_pb2.py
+*_pb2.py
```

Comment:
I actually changed it to this so that it would get some stray _pb2.py files in a rekall directory.  This picks up everything in all sub dirs.